Application/src/view/visuals/component/Message.java

```diff
@@ -231,11 +237,27 @@ public void setStatic(boolean staticIndicator)  {
     public void renderDefault(GraphicsContext gc) {
 
 
+        if(trails.size() != 0) {
+            Trail t;
+            //Draws all trails except the first and last one
+            for (int i = 1; i < trails.size() - 1; i++) {
+                // Points to the first index of the trail array
+                t = trails.get(i);
+                gc.drawImage(trail, t.getXcoordinate(), (t.getYcoordinate() + 18), t.getWidth(), t.getHeight());
```

Comment:
Again, why does it say that this has been changed? It looks exactly the same as the code I wrote

Reply:
Are you sure about that? Look at what this version of the for loop skips

Reply:
@IsabelleTornqvist that is how i fixed the bug, notice the "trails.size() - 1" which is looping through all indexes except the last one so we can add the trail
i made this comment already but it disappeared because i pushed changes, click on "show outdated" to see history of old comments

Reply:
I mean just line 246 247

Reply:
yes it exactly like yours, it is changed as a "new line" on git because it is in a new block of code. that is how git works

Reply:
To me it just looks like you "changed" my code, but didn't actually change anything

Reply:
@timjon Maybe look *where I added my comment. Im not talking about the "for (int i = 1; i < trails.size() - 1; i++) {" part

Reply:
again, the actual line wasn't changed, but it says "changed" on github, because it is not in the same place as before, that is how git works

Reply:
nej, that is not how git works. For example, the line "int x1 = this.node1.getX();" was prev. on line 240, but is now on 257, and that is not marked green/red as changed

Reply:
by "same place" my friend, i don't mean the line number, i mean the line was place in "another chunck of code", "another block of code", "another loop"
for the record. the amount of lines or "additions" you contributed to the project is not affected by my change if that is what is you are worried about

Reply:
I'm not concerned about "my contribution" in terms of lines of code, I'm concerned about traceability

Reply:
ok, then can you unblock this, because this is very time consuming

Reply:
Not sure what you mean. I'm reviewing your code. It's supposed to take time?

Reply:
Might be because its now inside of an if statement so the indentation is accually changed

Reply:
Yes code reviews supposed to take time, but i think this exact issue you are talking about is time consuming and i rather work on something else in the release week of the project!
at this stage, please provide test cases of my code, and point out any potential bugs or inconsistencies if they exist. 
same thing to @timjon  please unblock if the only issue you can see right now is only indentation or "changes on git"

Reply:
I understand what you are saying, and I agree that time should be spent reasonably. However, how would I know that this isn't a bigger issue than " just git-issues" if I don't spend time asking the questions? For me to have reached this understanding, this time was necessary.

Reply:
@IsabelleTornqvist fair enough

Reply:
+ for reference: it hasn't been more than 2 hours, and I have provided you at least some - in my opinion - valuble feedback.

Reply:
@IsabelleTornqvist yes you did, thanks